packages/web/resolvers/user.ts

```diff
@@ -63,6 +63,7 @@ const User = objectType({
     t.model.createdAt()
     t.model.membershipSubscription()
     t.model.isStudent()
+    t.model.configuration()
```

Comment:
This will expose this field publicly for everyone. I know giving up "what is this person's email digest preference" isn't exactly sensitive info, but it just doesn't quite feel right to make settings public record. You could use a similar pattern to email to do an auth check here.

Reply:
Done!